lib/unit-ruby/application_form.rb

```diff
@@ -1,18 +1,22 @@
 module Unit
   class ApplicationForm < APIResource
     path '/application-forms'
+    header 'X-Accept-Version' => 'V2024_06'
```

Comment:
I expect that at some point in the future this will become the default, at which point we should be able to remove this. I added this directly to the resource instead of allowing it to be passed in because 1) it's a lil cleaner, and 2) it seems better to enforce a breaking change like this at the library level instead of letting library consumers set custom headers on requests.